src/commcare_cloud/ansible/roles/elasticsearch/tasks/main.yml

```diff
@@ -231,3 +231,28 @@
     mode: 0640
   notify: reload monit
   tags: monit
+
+- name: Copy elasticsearch log_retention file
+  template:
+    src: "delete_older_es_logs.sh.j2"
+    dest: "/usr/local/sbin/delete_older_es_logs.sh"
+    group: root
+    owner: root
+    mode: 0700
+    backup: yes
+  when: elasticsearch_delete_old_logs|bool
+  tags:
+    - cron
+
+- name: Create Daily Cron job to delete elasticsearch_logs
+  cron:
+    name: "delete older elasticsearch logs"
+    job: "/usr/local/sbin/delete_older_es_logs.sh {{ elasticsearch_log_retention_days }}"
+    minute: 20
+    hour: "{{ nadir_hour|default(0) }}"
+    weekday: "1,2,3,4,5,6"
+    user: root
+    cron_file: delete_eslogs
```

Comment:
nit: sorry I should have pointed this one out to. Consider changing to the more consistent/specific `delete_older_es_logs`

Reply:
@dannyroberts I noticed it and changed yesterday to `delete_es_logs`, will modify it to `delete_older_es_logs`

```
(staging) ansible@es6-staging:/etc/cron.d$ ls -lrt
total 20
-rw-r--r-- 1 root root 396 Dec 23  2017 sysstat
-rw-r--r-- 1 root root 589 Jan 14  2020 mdadm
-rw-r--r-- 1 root root 190 Jun  5  2020 popularity-contest
-rw-r--r-- 1 root root 563 May 17 14:28 awsagent-update
-rw-r--r-- 1 root root 110 Jun 24 06:22 delete_es_logs
```

Reply:
Great thanks